jgiven-core/src/test/java/com/tngtech/jgiven/report/asciidoc/AsciiDocReportGeneratorTest.java

```diff
@@ -0,0 +1,76 @@
+package com.tngtech.jgiven.report.asciidoc;
```

Comment:
Shouldn't we at least have some test here that checks that we have actually generated a report?

Reply:
There is the identical named `jgiven-tests/src/test/java/com/tngtech/jgiven/report/asciidoc/AsciiDocReportGeneratorTest.java` which checks that the five meta files and one feature file is generated. Which was enough for me. Or to be honest I didn't want to handle files and file contents in the tests.